build/pipeline-releasebuild.yaml

```diff
@@ -5,14 +5,7 @@
  
 trigger: none
 pr: none
- 
-# Create a daily midnight build for release builds on master to ensure our release builds function
-schedules:
-- cron: "0 0 * * *"
-  displayName: Daily midnight build
-  branches:
-    include:
-    - master
+
```

Comment:
Thanks @bgavrilMS 
can we know about why we do that?

Reply:
This is a non-compliant release build. It will stop working very soon. If you want to keep it around for reference, keep the file. It would be better to delete it fully. 

If this build is accidentally used to release binaries, it will cause an icm.

Reply:
I like that it produces nightly bits, which the other does not do

Reply:
https://identitydivision.visualstudio.com/IDDP/_build/results?buildId=1317081&view=results - looks like OneBranch build does run nightly and produces nightly bits, which are also compliant.